requirements.txt

```diff
@@ -9,3 +9,4 @@ nvgpu
 psutil
 scipy >= 1.5.0, < 1.11.0
 termcolor
+diffrax<=0.4.0
```

Comment:
Keep the requirements list in alphabetical order. 

Maybe we could avoid the runtime warnings by changing the version of `diffrax`?